config/images/images.yaml

```
@@ -133,6 +133,7 @@ images:
   tags:
   - v2.6.1
   - v2.7.0
+  - v2.7.1
```

```suggestion
  - v2.7.1
```
We do not actually use the `v2.7.1` one, so I think it can be removed.

---

we can remove the v2.7.0?